spec/unit/provider/package/openbsd_spec.rb

```diff
@@ -46,6 +46,8 @@ def expect_pkgadd_with_env_and_name(source, &block)
     allow(described_class).to receive(:command).with(:pkginfo).and_return('/bin/pkg_info')
     allow(described_class).to receive(:command).with(:pkgadd).and_return('/bin/pkg_add')
     allow(described_class).to receive(:command).with(:pkgdelete).and_return('/bin/pkg_delete')
+
+    allow(Puppet::FileSystem).to receive(:exist?)
```

Comment:
Do we know who calls `Puppet::FileSystem.exists?` seems a bit weird to have this here.

Reply:
It's being stubbed on line 8, but due to adding `confine :feature => :hocon` in the puppetserver_gem provider, the method somehow gets called with `hocon.rb` and rspec complains since it's not what it expects.

As to why it gets called when executing `provider.install` I have no idea. My guess is it tries to find suitable providers... but the provider is already predetermined?

Reply:
Yeah, seems that gets triggered through facter 4 when puppet checks that the provider is suitable.